open-api/rest-catalog-open-api.yaml

```diff
@@ -1441,6 +1435,34 @@ components:
   #############################
   responses:
 
+    RESTCatalogConfigResponse:
+      description:
+        Configuration from the server consists of two sets of key/value pairs.
+        * defaults -  properties that should be used as default configuration; applied before client configuration
+        * overrides - properties that should be used to override client configuration; applied after defaults and
+                      client configuration
+        The REST Catalog configuration is constructed by setting the defaults, then client-
+        provided configuration, and finally overrides. The resulting final property set is then
+        used to configure the catalog.
+        
+        For example, a default configuration property might set the size of the
+        client pool, which can be replaced with a client-specific setting. An
+        override might be used to set the warehouse location, which is stored
+        on the server rather than in client configuration.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/RESTCatalogConfig'
```

Comment:
Will probably update to inline this and get rid of the `RESTCatalogConfig` object.